llama-index-integrations/vector_stores/llama-index-vector-stores-clickhouse/pyproject.toml

```diff
@@ -7,6 +7,11 @@ check-filenames = true
 check-hidden = true
 skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
 
+[tool.llamahub]
```

Comment:
Sorry lol small adjustment (still figuring out the structure we need for llamahub)

Lets do this

```
[tool.llamahub]
contains_example = false
import_path = "llama_index.vector_stores.clickhouse"

[tool.llamahub.class_authors]
ClickHouseVectorStore = "ClickHouse"
```